src/style.css

```diff
@@ -0,0 +1,26 @@
+body {
+  margin: 20px;
```

Comment:
As for me, better to use this margin in block.
If you had more elements on the page and need to move only few, this code will not help with this task.